check_process

```diff
@@ -20,7 +20,7 @@
         multi_instance=1
         incorrect_path=1
         port_already_use=0
-        change_url=1
+        change_url=0
```

Comment:
Why !?

Reply:
Package_check on master branch: change_url test already failed.
Testing manually: changing domain and/or path works

I think the issue is more from the change_url test.... I haven't took the time to look at it but change_url is all the time failed for packages that need to be at the root of the domain...